tests/test_app.py

```diff
@@ -1,4 +1,193 @@
-import todo_app
+import pytest
+from todo_app.models.view_model import ViewModel
+from todo_app.data.trello_items import Item
+from datetime import datetime, timedelta
+from dotenv import load_dotenv, find_dotenv
 
-def test_thing():
-    assert True
\ No newline at end of file
+file_path = find_dotenv('.env.test')
+load_dotenv(file_path, override=True)
```

Comment:
This is fine for what you have, but note that this is only running once at the start of the whole test run. If one test changed an environment variable to test a specific scenario, it would accidentally affect all subsequent tests. Better would be to make this a fixture and run it before every test.

Reply:
Resolved in module_4 https://github.com/ianm1968/ToDoTake2/commit/10760d33ef90f88f52dcb05aac6db3d48578f041

Reply:
Resolved in module_4 https://github.com/ianm1968/ToDoTake2/commit/10760d33ef90f88f52dcb05aac6db3d48578f041